dcm4che-core/src/main/java/org/dcm4che3/data/Attributes.java

```diff
@@ -753,7 +753,7 @@ public Sequence getSequence(String privateCreator, int tag) {
             
             return vr.toBytes(value, getSpecificCharacterSet(vr));
         } catch (UnsupportedOperationException e) {
-            LOG.info("Attempt to access {} {} as bytes", TagUtils.toString(tag), vr);
+            LOG.trace("Attempt to access " + TagUtils.toString(tag) + " " + vr + " as bytes",e);
```

Comment:
You can leave the 'e' and use the {} in log4j.

Reply:
I don't think the stack from the exception gets logged correctly using the {}.

Reply:
Craig is right. If you do it like this:
`LOG.info("Attempt to access {} {} as bytes", TagUtils.toString(tag), vr, e);`
The stack trace should get correctly logged.